docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -595,24 +595,40 @@ FileSystem.getTotalDiskCapacityAsync().then(totalDiskCapacity => {
 
 Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes, or JavaScript's [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) if the capacity is greater than 2<sup>53</sup> - 1 bytes.
 
+### `FileSystem.askForDirectoryPermissionsAsync(initialFileUrl)`
+
+**Android only**. Allows users to select a specific directory, granting your app access to all of the files and sub-directories within that directory.
+
+#### Arguments
+
+- **initialFileUrl (_string_)** -- **Optional**. The [SAF URI](#saf-uri) of the directory that the file picker should display when it first loads. **Available only on Android R or higher**.
+
+#### Returns
+
+Returns a Promise that resolves to a `false` if permissions weren't granted or `string` that represents a [SAF URI](#saf-uri).
```

Comment:
maybe we should follow something closer to our standard permission response format:

```
{
  status: 'granted' | 'denied';
  granted: boolean;
  directoryUri: string; // or directoryUrl? 👀 
}
```

Reply:
The return type was changed, but it doesn't include the `status`  key. I don't like the idea behind the strings comparison here.